app/templates/auth/change-password.html

```diff
@@ -0,0 +1,122 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Change password – Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": "/",
+        "label": "Digital Marketplace"
+      },
+      {
+        "link": dashboard_url,
+        "label": "Your account"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+
+{% if form.errors %}
+    <div class="validation-masthead" aria-labelledby="validation-masthead-heading">
+        <h3 class="validation-masthead-heading" id="validation-masthead-heading">
+            There was a problem with the details you gave for:
+        </h3>
+        <ul>
+        {% for field_name, field_errors in form.errors|dictsort|reverse if field_errors %}
+          {% for error in field_errors %}
+            <li><a href="#{{ form[field_name].name }}" class="validation-masthead-link">{{ form[field_name].label.text }}</a></li>
+          {% endfor %}
+        {% endfor %}
+        </ul>
+    </div>
+{% endif %}
+
+<header class="page-heading">
```

Comment:
Could you use a page heading from the tookit? http://alphagov.github.io/digitalmarketplace-frontend-toolkit/page-heading.html